Sources/Workspace/Workspace.swift

```diff
@@ -928,7 +933,8 @@ public class Workspace {
         let dependencies = transitiveClosure(rootManifests.map{ KeyedPair($0, key: $0.url) }) { node in
             return node.item.package.dependencies.flatMap{ dependency in
                 // Check if this dependency is available.
-                guard let managedDependency = managedDependencies[RepositorySpecifier(url: dependency.url)] else {
+                let repository = RepositorySpecifier(url: dependency.url)
+                guard let managedDependency = managedDependencies[repository] else {
```

Comment:
managed dependencies has a string subscript overload so constructing RepositorySpecifier object can be avoided here.